src/applications/personalization/dashboard/components/health-care-v2/HealthCareContentV2.jsx

```diff
@@ -239,22 +239,20 @@ const mapStateToProps = state => {
     'Roseburg (Oregon) VA health care',
     'White City health care',
   ];
-  if (state?.featureToggles?.pwEhrCtaDrupalSourceOfTruth) {
```

Comment:
so this is not checking for the feature toggle at all anymore? is this because this is the health-care-v2 vs v1?

Reply:
well that and the cerner team told me we won't need it. They aren't turning the flag off. If the flag is off the drupal connection still works. The drupal thing is permanent now. I assume it will be removed down the line

Reply:
if it somehow breaks between now and then we will likely turn off healthcare v2